README.md

```diff
@@ -1,44 +1,26 @@
-=== SureAliveD ===
+# SureAliveD
 
-1. What is SureAlived?
+## What is SureAliveD?
 
-SureAliveD is a very efective LVS real service tester. We split
-it on two separated programs - surealived (tester) and ipvssync
-(kernel ipvs table synchroniser). 
-We deceide to replace keepalived tester functionality (not VRRP)
-- it has some bugs and a lack of some features we would like to have.
+SureAliveD is a very effective LVS service tester that is meant to replace `keepalived` (but not VRRP). Application is actually two separated programs:
```

Comment:
splitted to two seperated programs
I think, that sounds better

Reply:
We could split something that was joined, but here it's not the case ;P. 

btw past participle form of a verb split is split.